snapcraft/project/_project_info.py

```diff
@@ -86,4 +86,7 @@ def _load_yaml(*, yaml_file_path: str) -> OrderedDict:
     except yaml.YAMLError as e:
         raise errors.YamlValidationError(str(e), yaml_file_path) from e
 
+    if yaml_contents is None:
```

Comment:
Can you add a unit test for this, one that of an empty file and validate that the instantiation of a ProjectInfo returns the correct YamlValidationError?

Reply:
Unit test added.